### DISCUSSION TITLE
Validate File/Image Uploads

### DISCUSSION THREAD

**Discussion by eddycodes** [skydiver/october-plugin-forms#243] [General]
How do you validate file/image uploads? I checked through the documentation and I could not find a helpful article.

**Comment by snipiba** [comment]
i tried everything.. fileupload, fileuplader, uploader, _uploader, file, files... no way to set file upload working with validation like required, mimetypes etc. :/ some stuff in documentation is mandratory

**Comment by eddycodes** [comment]
I feel validation for file uploads is not supported. It seems @skydiver hasn't updated the project in a long time. I'm worried we will not see this feature
